src/aliases/,aliases_ram_lowmem.s

```diff
@@ -63,7 +63,7 @@
 	!addr FAC1_mantissa   = $62 ; $62 - $65
 	!addr FAC1_sign       = $66
 
-	!addr SGNFLG    = $67 ;          -- NOT IMPLEMENTED --
+	!addr DEGREE    = $67 ;          Number of terms of series evaluation
```

Comment:
Please keep the address name - `SGNFLG` is a canonical name of this memory location, see:
- https://github.com/mist64/c64ref/blob/main/src/c64mem/c64mem_jb.txt
- https://legacyblog.citizen428.net/studynotes/c64/
- http://www.cbmhardware.de/show.php?r=4&id=13/Commodore%20plus/4%20and%20Commodore%2016%20Memory%20Map

or the _Mapping The C64_ book.

Reply:
Ok, I'll change back. One source gave dual names, which is why I changed.

Reply:
Changed back. Using `SGNFLG` now.